stream-chat-android-ui-components/src/main/res/layout/stream_ui_attachment_gallery_options_view.xml

```diff
@@ -49,7 +49,7 @@
             />
 
         <TextView
-            android:id="@+id/saveImage"
+            android:id="@+id/saveMedia"
             style="@style/StreamUiMessageOptionItem"
             android:text="@string/stream_ui_attachment_gallery_save_image"
```

Comment:
Should this be "Save media", or "Save image" / "Save video" depending on the selected page?

Reply:
fixed